content/posts/ifk-smhi-2023-03-29.md

```diff
@@ -15,7 +15,7 @@ har föreningen, främst genom dr. Mladen Gibanicas försorg, arbetat med att ta
 fram ett pythonbibliotek för att på ett enkelt sätt läsa data från SMHIs API.
 Söndagen den 27 mars släpptes den första officiella versionen, och nu finns
 version 0.1.1 tillgänglig via paketinstallationsplattformen
-<a href=""https://pypi.org/project/ifk-smhi/" target="_blank">pypi</a>.
+<a href=""<https://pypi.org/project/ifk-smhi/>" target="_blank">pypi</a>.
```

Comment:
Don't know what happened here, any suggestions @mgcth ?

Reply:
I think you should remove <> and the extra ".